code/modules/shuttle/ert.dm

```diff
@@ -5,6 +5,14 @@
 	possible_destinations = "specops_home;specops_away;specops_custom"
 	resistance_flags = INDESTRUCTIBLE
 
+/obj/machinery/computer/shuttle/ert/Topic(href, href_list)
+	if(href_list["move"])
+		var/authorized_roles = list(SPECIAL_ROLE_ERT, SPECIAL_ROLE_DEATHSQUAD)
+		if(!((usr.mind.assigned_role in authorized_roles) || is_admin(usr)))
```

Comment:
More a design point:
This would mean that admins themselves as characters won't trigger the 'alarm'. If this is undesired then a simple check on the mobtype of the users mob will fix it. (aghosting etc)

Reply:
Yeah that is intended behaviour. If an admin spawns in and takes the shuttle there or back, he doesn't want an alert. And if an admin as normal station crew takes the shuttle back to CC, then he's already aware of what's going on.